### PR TITLE
add sticky prop to docs

### PR DESCRIPTION
**Pull request by britt6612**
Add `sticky` prop to docs

### CODE REVIEW

File: src/screens/Header.js [hunk 1]
```diff
@@ -38,6 +44,16 @@ const HeaderPage = () => (
         },
       }}
     />
+
+    <Property name="sticky">
```
Comment (jcfilben) [inline src/screens/Header.js:48]: Need to wrap `Property` with `Properties`
```suggestion
  <Properties>
    <Property name="sticky">
```

Comment (britt6612) [reply src/screens/Header.js]: done

File: src/screens/Header.js [hunk 2]
```diff
@@ -38,6 +45,17 @@ const HeaderPage = () => (
         },
       }}
     />
```
Comment (jcfilben) [inline src/screens/Header.js:47]:
```suggestion
    >
```

File: src/screens/Header.js [hunk 3]
```diff
@@ -38,6 +45,17 @@ const HeaderPage = () => (
...
+      <Property name="sticky">
+        <Description>
+          Whether the Header should hide when the user is scrolling down and
+          appear once the user scrolls back up.
+        </Description>
+        <PropertyValue type="string">
+          <Example>`sticky= "scrollUp"`</Example>
+        </PropertyValue>
+      </Property>
+    </Properties>
```
Comment (jcfilben) [inline src/screens/Header.js:58]:
```suggestion
    </Properties>
    </ComponentDoc>
```

**Review by jcfilben — COMMENTED** [review]
Sorry I missed this in my previous review but `ComponentDoc` should wrap `Properties`

**Comment by britt6612** [comment]
> Sorry I missed this in my previous review but `ComponentDoc` should wrap `Properties`

got it okay Ill change

**Review by jcfilben — APPROVED** [review]
Looks good!

**Pull request merged**